e20-kit3-demo/snapconnect-exosite-example/main.py

```diff
@@ -0,0 +1,100 @@
+import os
+import sys
+import json
+import binascii
+
+from snapconnect import snap
+from apy import ioloop_scheduler
+
+import tornado.ioloop
+from tornado import httpclient
+
+from pyonep import onep  # Exosite Python Library
+
+
+# TODO: Replace these with values from your own exosite account and resource
+# We like to map SNAP addresses to Exosite CIKs
+EXOSITE_CIKS = {"000020": ''}
```

Comment:
Need a better explanation comment. For example there is only one CIK per gateway (?). And show an example of CIK format XXX:XX:XXNN, etc.  Also, on a real GW we'll need to dynamically get the SNAP address, since it's automagically retrieved from MAC addr.